### ISSUE TITLE
Inconsistent naming: organization.organisationType

### ISSUE THREAD

**Issue by drkane** [ThreeSixtyGiving/standard#184]
The `organization.organisationType` should be `organization.organizationType` for consistent spelling. 

Apart from this case the schema seems to consistently use "organization" in field names and "organisation" in descriptions.

Would presumably break backwards-compatability.

**Comment by caprenter** [comment]
Thanks @drkane 

Without looking into this in great depth, yeah this would appear to be a little annoying. However, as field names are just field names, it shouldn't really matter (although obviously it could easily trip people up creating and using data)

You're right tho, that a change would not be backwards compatible, so it's unlikely that this will get changed anytime soon. 

You may know better than me how extensively that field is being used, so we might be able to argue that a change sooner rather than later is a good thing.

**Comment by drkane** [comment]
Thanks - on a quick look I couldn't find anyone using this field in existing publishers, although Big Lottery fund have a non-standard field called `BIGField_Organisation_Type` that effectively is the same.

This may be something that's for the validation tools - as you say I can imagine lots of different combinations of `s`/`z` spellings for these fields. I don't know if there's scope in the schema for allowing `organisation` as an alternative spelling for `organization` and indicating to users that they should be treated as the same by anything that uses the data.

One other option would be just to call it `organization.type` - this would be consistent with `organization.name`, etc, although I guess the reasoning is that the meaning of "type" could be ambiguous.

**Comment by timgdavies** [comment]
Good idea on ```organization.type```, particularly if we allow the value to be an array.

**Comment by stevieflow** [comment]
There's a link #104 (z vs s)
